### PR TITLE
11765 add metadata to plots

### PR DESCRIPTION
**Pull request by wenebob**
This pull request is to add metadata to slope plot (from issue #11765 )
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: examples/basic/annotations/slope.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+''' A marker plot that demonstrates a slope.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.slope, bokeh.plotting.figure.circle
+    :refs: :ref:`ug_basic_annotations_slopes`
```
Comment (bryevdv) [inline examples/basic/annotations/slope.py]: the existing ref is not pluralized
```suggestion
    :refs: :ref:`ug_basic_annotations_slope`
```

**Review by bryevdv — COMMENTED** [review]
Just one small comment

**Comment by wenebob** [comment]
okay @bryevdv. I have committed your suggestion. Thank you for the correction  💯

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Faith-Nchifor !

**Pull request merged**